accumulo/src/main/java/org/vertexium/accumulo/keys/DataTableRowKey.java

```diff
@@ -41,7 +44,10 @@ public String getPropertyKey() {
         return parts[PARTS_INDEX_PROPERTY_KEY];
     }
 
-    public long getPropertyTimestamp() {
-        return Long.parseLong(parts[PARTS_INDEX_PROPERTY_TIMESTAMP]);
+    public static boolean isLegacy(String dataRowKey) {
```

Comment:
@mwizeman based on problems I had on real data I need to check the following and skip the timestamp limiting above.